code/modules/urist/gamemodes/paranoia/conspiracyitems.dm

```diff
@@ -12,4 +12,47 @@
 /obj/item/weapon/reagent_containers/glass/beaker/vial/random/jetfuel
 	random_reagent_list = list(
 		list("fuel" = 15, "thermite" = 15)	= 9,
-		list("fuel" = 30)	 = 1,) //10% chance, the mix cannot, in fact, melt steel beams.
\ No newline at end of file
+		list("fuel" = 30)	 = 1,) //10% chance, the mix cannot, in fact, melt steel beams.
+
+/obj/item/weapon/reagent_containers/glass/beaker/vial/random/jetfuel/New()
+	..()
+	desc = "Contains jet fuel. Warning: results may vary!"
```

Comment:
If that's a serious question, I don't know, it's literally thermite doing the trick.
If not, see above - 10% chance that JET FUEL CANNOT MELT STEEL GIRDERS.